src/main/java/com/minecolonies/client/gui/WindowTownHall.java

```diff
@@ -178,6 +239,11 @@
     private List<CitizenDataView>   citizens    = new ArrayList<>();
 
     /**
+     * List of workOrders.
+     */
+    private List<WorkOrderView> workOrders = new ArrayList<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private field 'workOrders' could be made final; it is only initialized in the declaration or constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AImmutableField)